README.md

```diff
@@ -118,6 +120,17 @@ The backend system would collect all `sensu_check` resources.
   Sensu_check <<||>>
 ```
 
+### Resource purging
+
+All the types provided by this module support purging except `sensu_config`.
+This example will remove all unmanaged Sensu checks:
+
+```puppet
+resources { 'sensu_check':
```

Comment:
`sensu_asset` should be called out in the docs. We should also link to the ticket in sensu-go to get this to work or where they say they are not going to support it.